src/etc/lldb_providers.py

```diff
@@ -32,7 +32,7 @@
 # You can find more information and examples here:
 #   1. https://lldb.llvm.org/varformats.html
 #   2. https://lldb.llvm.org/use/python-reference.html
-#   3. https://lldb.llvm.org/python_reference/lldb.formatters.cpp.libcxx-pysrc.html
+#   3. https://cgit.geodns-americas.gentoo.org/fork/llvm-project.git/plain/lldb/www/python_reference/lldb.formatters.cpp-pysrc.html?h=release/3.8.x
 #   4. https://github.com/llvm-mirror/lldb/tree/master/examples/summaries/cocoa
```

Comment:
This should probably be changed to point to the official llvm-project repo instead.